### ISSUE TITLE
Issue with logging in container

### ISSUE THREAD

**Issue by IlyaVorozhbit** [n1215/roadrunner-docker-skeleton#10]
After installing it (task init, task up)
```
ilya@DESKTOP-2U45ORR:~/work/roadrunner-docker-skeleton$ ./task login
Attempt to login roadrunner container...

OCI runtime exec failed: exec failed: container_linux.go:380: starting container process caused: exec: "bash": executable file not found in $PATH: unknown
```

**Comment by IlyaVorozhbit** [comment]
It was in WSL 2, Ubuntu 20.04

**Comment by n1215** [comment]
@IlyaVorozhbit 
Thank you for your report.
I fixed the bug in https://github.com/n1215/roadrunner-docker-skeleton/pull/11

**Comment by n1215** [comment]
Thank you for your report.
I fixed the bug in #11.
